sandbox/id-authentication-partner.properties

```diff
@@ -0,0 +1,2 @@
+ida-websub-masterdata-templates-callback-url=${mosipbox.public.url}${server.servlet.context-path}/callback/masterdata/templates
```

Comment:
can we rename the property file as id-authentication-external.properties?

Reply:
done